lib/view_model.rb

```diff
@@ -212,6 +227,10 @@ def visible!(context: self.class.new_serialize_context)
     end
   end
 
+  def context_for_update(deserialize_context:)
```

Comment:
Please explain this in a comment: what kind of update is the context used for, when might you override this?